spec/paypalhttp/encoder_spec.rb

```diff
@@ -116,13 +116,46 @@
           "content-type" => "application/x-www-form-urlencoded; charset=utf8"
         },
         :body => {
-          :key => " ..<..>..%..{..}..|../..^..`..!",
+          :key => " ..<..>..%..{..}..|..^..`",
```

Comment:
The characters we wanted to remove from this test were ":" and "/", why was "!" removed from the character list